src/windows/users.rs

```diff
@@ -99,6 +99,8 @@ pub unsafe fn get_users() -> Vec<User> {
                 if (*buf).usri1_flags & UF_NORMAL_ACCOUNT != 0 {
                     let groups = get_groups_for_user((*buf).usri1_name);
                     users.push(User {
+                        uid: (*buf).usri1_user_id,
+                        gid: 0,
```

Comment:
There is no GID on windows?

Reply:
Not that I could find.  I can dig a bit more if you would like

Reply:
So after digging a bit, I found information on Windows.  They don't have an individual group assigned to the user.   They do however have unique [Security Identifiers](https://docs.microsoft.com/en-us/windows/win32/secauthz/searching-for-a-sid-in-an-access-token-in-c--) made up of various [Components](https://docs.microsoft.com/en-us/windows/win32/secauthz/sid-components).   This may be a candidate for this  field, but it doesn't map well to group id.  Also retrieving this would probably out of the scope of this PR.

Reply:
Please add those explanations on the getter for the group id on the `UserExt` trait then. Not the first weird thing from windows...

Reply:
Updated.